t/page/country.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/country'
+require 'pry'
+
+describe 'Country' do
+  let(:all_countries) do
+    [
+      {
+        name: 'Abkhazia',
+        url:  'abkhazia',
+      },
+    ]
+  end
+
+  it 'detects that a country is not missing' do
+    page = Page::Country.new('abkhazia', all_countries)
+    refute_nil page.country
```

Comment:
Presumably you'd quite like it to get the correct country here, rather than just anything that's non nil?